app/javascript/components/menus/index/menus-container.vue

```diff
@@ -47,11 +47,12 @@
         >
           {{ $t('msg.noElements') }} {{ $t('msg.menus.title').toLowerCase() }}
         </p>
-        <menus-table
-          v-else
+        <div v-else class="flex w-full 2xl:justify-center items-center overflow-auto">
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'class' should be on a new line                        vue/max-attributes-per-line